CHANGELOG.md

```diff
@@ -0,0 +1,480 @@
+- Add stable URLs for package resources in yarn workspace packages using the package.json name
+
+## 4.1.0 - 2019-03-20
+
+### Changed
+- Switched from awesome-typescript-loader to ts-loader + fork-ts-checker-webpack-plugin + tsconfig-paths-webpack-plugin
+
+## 4.0.12 - 2019-02-06
+
+### Fixed
+- Revert acorn change in 4.0.11
+
+## 4.0.11 - 2019-02-05
+
+### Fixed
+- Hard coded the acorn dependency to less than 6.0.7, as it breaks builds with webpack
+
+## 4.0.10 - 2018-12-14
+
+### Fixed
+- Fixed double encoding issue with html diffs.
+- Fixed issue where IE would throw split of undefined when reporting errors since trace was sometimes an object.
+
+## 4.0.9 - 2018-12-12
+
+### Fixed
+- Fixed encoding issue of raw error messages the if the error message contained html it would be rendered on page.
+
+## 4.0.7 - 2018-12-03
+
+### Removed
+- Removed bedrock-repl bin from package.json
+
+## 4.0.6 - 2018-12-03
+
+### Added
+- Added startup message listing version for quicker feedback
+
+### Improved
+- Improved over-escaping of error messages
+
+### Removed
+- Removed bedrock-repl as it didn't make sense without bolt
+- Removed all dependence on bolt
+
+## 4.0.5 - 2018-11-30
+
+### Fixed
+- Fixed diff error message reporting
+
+## 4.0.3 - 2018-11-30
+
+### Fixed
+- Gracefully timeout individual tests
+
+## 4.0.0 - 2018-11-23
+
+### Added
+- Test results are sent to node server instead of being scraped using selenium
+- Tests are chunked into batches to 'solve' memory leaks
+- Tests are retried once on failure (when not stopping)
+- Testing page is reloaded on failure to avoid cascades
+
+### Changed
+- Test HUD has been reformatted
+
+### Removed
+- Removed SauceLabs integration
+
+## 3.2.1 - 2018-10-30
+
+### Added
+- Add error content to the UnitTest logging output
+
+## 3.2.0 - 2018-10-30
+
+### Added
+- Supports logging history in UnitTest
+
+## 3.0.0 - 2018-08-24
+
+### Changed
+- Upgraded to typescript 3
+
+## 2.4.0 - N/A
+
+### Added
+- added support for shift key in key combos
+
+## 1.8.0 - 2017-11-09
+
+### Added
+- support for chrome and firefox headless modes in bedrock-auto via `chrome-headless` and `firefox-headless`
+
+## 1.7.0 - 2017-07-28
+
+### Added
+- added IFrameTest.js to the samples
+- introduced '=>' symbol in selectors to represent inside an iframe's contents
+- added --delayExit command line option to keep browser open after finishing test run
+
+### Changed
+- updated key effects and mouse effects to handle dispatching within iframes
+- changed click from an ActionSequence to a WebElement action because Edge supports it
+
+
+## 1.4.0 - 2016-12-09
+
+### Added
+- adds testing page for scraping content
+- generates reports on timeouts
+
+### Changed
+- uses getText instead of getInnerHtml to match new selenium webdriver APIs
+
+### Improved
+- improves test report format
+- improves experimental SauceLabs integration
+
+
```

Comment:
A whole bunch of newlines?

Reply:
Also like this previously. I've fixed it.